src/containerapp/azext_containerapp/_params.py

```diff
@@ -394,6 +394,10 @@ def load_arguments(self, _):
         c.argument('consumer_secret_setting_name', options_list=['--consumer-secret-name', '--secret-name'], help='The consumer secret name that contains the app secret.')
         c.argument('provider_name', required=True, help='The name of the custom OpenID Connect provider.')
         c.argument('openid_configuration', help='The endpoint that contains all the configuration endpoints for the provider.')
+        c.argument('token_store', options_list=['--token-store'], arg_type=get_three_state_flag(), help='Boolean indicating if token store is enabled for the app.')
+        c.argument('sas_url_secret', options_list=['--sas-url-secret'], help='The blob storage SAS URL to be used for token store.')
+        c.argument('sas_url_secret_name', options_list=['--sas-url-secret-name'], help='The secret name that contains blob storage SAS URL to be used for token store.')
```

Comment:
It is no need to explicitly declare the default option name
```suggestion
        c.argument('token_store', arg_type=get_three_state_flag(), help='Boolean indicating if token store is enabled for the app.')
        c.argument('sas_url_secret', help='The blob storage SAS URL to be used for token store.')
        c.argument('sas_url_secret_name', help='The secret name that contains blob storage SAS URL to be used for token store.')
```